lib/matplotlib/patches.py

```diff
@@ -3916,6 +3927,7 @@ def __str__(self):
         return s % (self._x, self._y, self._width, self._height)
 
     @_docstring.dedent_interpd
+    @_api.make_keyword_only("3.6", name="bbox_transmuter")
```

Comment:
Probably start this at boxstyle since bbox_transmuter is deprecated (you could remove that here even?).

Reply:
Started one later with `mutation_scale`. Same argument with erring on the permissive side `FancyBboxPatch((0, 0), 5, 8, "round") is still sort of reasonable.